.ruby-gemset

```diff
@@ -0,0 +1 @@
+complexity-of-need
```

Comment:
It could be worth gitignoring this file, too. I'm not sure it's really needed as part of the project –  it's more about how developers choose to configure their local machine.